e2e/main_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2013 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package e2e
```

Comment:
i feel we should move the duplicate code (which you copied from integration) to pkg/testutil. Then we can reuse?